odata-protocol/11.4 Data Modification.md

```diff
@@ -159,7 +159,10 @@ related entity, those related entities MUST be included either as
 references to existing entities or as content for new related entities.
 
 An entity may also be created as the result of an
```

Comment:
Does this cover the case of a nullable singleton? Singletons are not entities, are they?

Reply:
This section is about POST to a collection URL. A singleton is _not_ a one-element collection, and OData does not define POST to a singleton URL. We had a long debate on this and decided against it.

So this sentence here does not apply to singletons. 

The referenced sentence on PUT/PATCH to a single entity URL does apply to singletons.